test/sql/aggregate/aggregates/test_quantile_list.test

```diff
@@ -0,0 +1,44 @@
+# name: test/sql/aggregate/aggregates/test_quantile_list.test
+# description: Test QUANTILE operator with LIST quantiles
+# group: [aggregates]
+
+statement ok
+PRAGMA enable_verification
+
+# scalar quantiles
+statement ok
+create table quantiles as select range r, random() from range(10000) union all values (NULL, 0.1), (NULL, 0.5), (NULL, 0.9) order by 2;
```

Comment:
Could you add a tests for quantiles on constant values (e.g. `SELECT quantile(1, LIST_VALUE(0.1, 0.5, 0.9)) FROM quantiles`)?